cocos/2d/CCParticleSystem.cpp

```diff
@@ -846,6 +848,8 @@ void ParticleSystem::update(float dt)
         _emitCounter -= rate * emitCount;
```

Comment:
If the count down cause _emitCounter to be negative, we should solve the issue while counting down